src/components/DatePicker/DatePicker.js

```diff
@@ -329,14 +329,26 @@ export default class DatePicker extends Component {
   };
 
   rightArrowHTML() {
-    return `
+    return componentsX
+      ? `
+      <svg width="16px" height="16px" viewBox="0 0 16 16">
```

Comment:
Does `carbon-elements` have a corresponding icon...? If so, would it be a option using React string renderer, or would it be a too much overhead for client-side codebase...?

Reply:
I believe the `carbon-elements` source has hard coded ID and class attributes, but I think we could use `@carbon/icon-helpers` to convert it to a string

Reply:
OK if we can do it build-time instead of browser run-time it'd be great, but we can do it after `v10`. Thank you for your insights @emyarod!